modules/xphoto/src/bm3d_image_denoising.cpp

```diff
@@ -264,38 +268,38 @@ void bm3dDenoising(
 #else
 
 void bm3dDenoising(
-    InputArray _src,
-    InputOutputArray _basic,
-    OutputArray _dst,
-    float h,
-    int templateWindowSize,
-    int searchWindowSize,
-    int blockMatchingStep1,
-    int blockMatchingStep2,
-    int groupSize,
-    int slidingStep,
-    float beta,
-    int normType,
-    int step,
-    int transformType)
+    InputArray /* _src */,
+    InputOutputArray /* _basic */,
+    OutputArray /* _dst */,
+    float /* h */,
+    int /* templateWindowSize */,
+    int /* searchWindowSize */,
+    int /* blockMatchingStep1 */,
+    int /* blockMatchingStep2 */,
+    int /* groupSize */,
+    int /* slidingStep */,
+    float /* beta */,
+    int /* normType */,
+    int /* step */,
+    int /* transformType */)
 {
     // Empty implementation
```

Comment:
To prevent user confusing please raise an error in this case:
```
CV_Error(CV_StsNotImplemented, "The algorithm is patented and is excluded in this configuration; Set OPENCV_ENABLE_NONFREE CMake option and rebuild the library");
```

P.S. There is macro CV_UNUSED: https://github.com/opencv/opencv/blob/5f30a0a076e57c412509becd1fb618170cbfa179/modules/core/src/stat.cpp#L1745